src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessaryParenthesesCheck.java

```diff
@@ -89,6 +89,15 @@
  * if ((++f) &gt; g &amp;&amp; a) { // violation, unnecessary paren
  * }
  * </pre>
+ * <p>
+ * Partial not supported for operator precedence includes cases of the following type:
```

Comment:
Probably a continuation of https://github.com/checkstyle/checkstyle/pull/12703#discussion_r1166953481

I am still not following what is "partially supported" and what is missing.

> I will change the new text some thing like The  partial support for operator precedence includes bitwise operator but they are not supported by default
is it fine

This is not what partial support means. As an analogy, you don't really say a car does not support driving until a key is turned in the ignition.

If you support everything your tokens does when enabled, then this paragraph needs to be removed. Otherwise, it needs to be updated specifically with what is and what is not supported.

https://github.com/checkstyle/checkstyle/pull/12703#discussion_r1167617023
> I would expect things like: `violation not reported because we are not type aware` and `violation reported but this could fail if X was Y`.

Reply:
>I am still not following what is "partially supported" and what is missing

I am sorry for this confusion.

> If you support everything your tokens does when enabled 

yes, in these three new tokens, support everything similar as https://docs.oracle.com/javase/tutorial/java/nutsandbolts/operators.html#:~:text=logical%20AND,%7C%7C
 in these three tokens, there is no operator associativity. that's why it is similar to logical or, logical and
 
 apart of this they wont support cases like https://checkstyle.org/config_coding.html#UnnecessaryParentheses:~:text=The%20check%20is%20not,if%20(p%20%3D%3D%20(q%20%3C%3D%20r))%20%7B%7D. 
 
 so i am removing the para

Reply:
@rnveach ping